subprojects/docs/src/samples/userguide/mavenMigration/basic/kotlin/build.gradle.kts

```diff
@@ -26,7 +26,7 @@ dependencies {
 
 // tag::pom-dependencies[]
 dependencies {
-    testImplementation("org.codehaus.groovy:groovy-all:2.5.4")
+    testImplementation("org.codehaus.groovy:groovy-all:2.5.7")
```

Comment:
I'd say we shouldn't upgrade samples versions randomly. Instead, we should tackle this: https://github.com/gradle/gradle/issues/6044

Reply:
Upgraded, to make sure they pass and that I don't leave them behind, i.e. create more chores.
Would you like me to revert these?

Reply:
I think it's okay either way. But we should push forward with doing this automatically.

Reply:
Absolutely!